src/styles/index.css

```diff
@@ -0,0 +1,106 @@
+body {
+  background-color: rgb(202, 201, 201);
+  font-family: "Roboto", sans-serif;
+}
+
+.main {
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.main__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.main__img {
+  background: url(../images/imac.jpeg) no-repeat;
+  margin: 0 auto 40px;
+  background-position: center;
+  width: 160px;
+  height: 134px;
+  background-size: contain;
+}
+
+.main__h1 {
+  color: var(--Main-accent, #060b35);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 150%;
+  margin: 0 0 4px;
+}
+
+.main__h2 {
+  color: var(--Secondary, #616070);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 140%;
+  margin: 0 0 16px;
+}
+
+.r {
+  color: var(--Main-accent, #060b35);
```

Comment:
I cannot find where you created these vars, but anyway, let's use scss variables in scss